resources/web/server/kill_old_webots_instances.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import psutil
+import re
+import time
+import os
+
+'''
+Kill all instances of streaming Webots processes fulfilling the following conditions:
```

Comment:
Is this script supposed to work on any platform? If not we should explain on which platforms it works.